pulp_docker/app/serializers.py

```diff
@@ -162,3 +164,88 @@ class Meta:
         model = models.DockerDistribution
         fields = tuple(set(RepositoryVersionDistributionSerializer.Meta.fields) - {'base_url'}) + (
             'registry_path',)
+
+
+class ManifestTagOperationSerializer(serializers.Serializer):
```

Comment:
do you think we can call it TagOperaionSerializer and TagImageSerializer, UnTagImageSerializer respectively?

Reply:
Sure, great idea.